### DISCUSSION TITLE
e.target?

### DISCUSSION THREAD

**Discussion by zachkobrinsky** [prabhuignoto/react-chrono#200] [Q&A]
Hi I'm not seeing an onClick for react-chrono, just an onItemSelected, but it only returns the item object from the items array. Is there a way to access the actual dom element?

**Comment by prabhuignoto** [comment]
Hi @zachkobrinsky , yes currently only the selected object is returned and there is no way to access the actual dom. This will be added to the next release 1.13. Thanks for the f/b
